pages/capitalization.md

```diff
@@ -15,6 +14,11 @@ capitalization guidelines:
 
 -   Don't capitalize “federal” or “government”
 
+When you're deciding whether to capitalize noun phrases, keep in mind that in English, title case is often a marker of formality. Using it judiciously can help clarify that you're speaking about a specific, official thing (such as a form, office, or person). Overuse can cause users stress by implying formality or officialness where it doesn't exist. For instance:
```

Comment:
Love this, @coreycaitlin! I agree with @tadhg-ohiggins's language comment, though "status" may be enough or "significance"...just a few ideas.